frontend/src/Editor/Container.jsx

```diff
@@ -313,7 +310,14 @@ export const Container = ({
       setBoxes(newBoxes);
       updateCanvasHeight(newBoxes);
     },
-    [isVersionReleased, enableReleasedVersionPopupState, boxes, setBoxes, selectedComponents, updateCanvasHeight]
+    [
+      isVersionReleased,
+      enableReleasedVersionPopupState,
+      boxes,
+      setBoxes,
+      useEditorStore.getState().selectedComponents,
```

Comment:
We're not re-rendering the components when the value is changed. So, this will not work what I feel. We can remove `useEditorStore.getState().selectedComponents` from the dependency as we're fetching the latest data inside the function. Check whether it is working after removing it and if so, we can proceed with that

Reply:
Yes, not needed,  removed it.